lib/block-supports/layout.php

```diff
@@ -334,10 +334,11 @@ function gutenberg_render_layout_support_flag( $block_content, $block ) {
 		$child_layout_styles = array();
 
 		if ( 'fixed' === $block['attrs']['style']['layout']['selfStretch'] && isset( $block['attrs']['style']['layout']['flexSize'] ) ) {
+			$has_shrink            = isset( $block['attrs']['style']['layout']['breakOutOfLayout'] ) && $block['attrs']['style']['layout']['breakOutOfLayout'];
```

Comment:
Tiny style nit (don't worry about updating this if you prefer the explicit approach):

Would it be worth using `_wp_array_get` here with a default value of `false`?

E.g. something like `_wp_array_get( $block, array( 'attrs', 'style', 'layout', 'breakOutOfLayout' ), false )`